XXXX-template.md

```diff
@@ -0,0 +1,60 @@
+---
+simd: 'XXXX'
+title: Title of SIMD
+authors:
+  - (fill in with names of authors)
+category: Standard/Meta
+type: Core/Networking/Interface/Meta
+status: Draft
+created: (fill me in with today's date, YYYY-MM-DD)
+feature: (fill in with feature tracking issues once accepted)
+---
+
+## Summary
+
+A brief summary of what the feature is.
+
+## Motivation
+
+Why are we doing this? What use cases does it support? What is the expected
+outcome?
+
+## Alternatives Considered
+
+What alternative designs were considered and what pros/cons does this feature
+have relative to them?
+
+## Detailed Design
+
+Explain the feature as if it was already implemented and you're explaining it
+to another Solana core contributor. The generally means:
```

Comment:
missing explanation?

Reply:
Meant to be writing the following sections, `New Terminology`, `Design`, and `Impact`. Good point though, having both may be confusing.

Reply:
Updated structure to make more sense